src/include/network/lan/bridge.rb

```diff
@@ -34,32 +34,97 @@ def initialize_network_lan_bridge(_include_target)
       textdomain "network"
     end
 
+    # It returns an empty string if given IP is 0.0.0.0, otherwise returns
+    # the given IP
+    #
+    # @param [String] ip
+    # @return [String] empty string if given ip is 0.0.0.0; given ip
+    # otherwise
+    def bridge_ip(ip)
+      ip == "0.0.0.0" ? "" : ip.to_s
+    end
+
+    # Checks if the given interface is using the old way of config enslaved
+    # interfaces with bootproto as static and 0.0.0.0 as IPADDR
+    #
+    # @param [String] interface name
+    # @return [Boolean] returns true if given enslaved interface is configured
+    # in the old way
+    def old_bridge_port_config?(ifcfg_name)
+      devmap = LanItems.GetDeviceMap(LanItems.find_configured(ifcfg_name))
+      return false unless devmap
+
+      devmap["BOOTPROTO"] == "static" && devmap["IPADDR"] == "0.0.0.0"
+    end
+
     # Immediately updates device's ifcfg to be usable as bridge port.
     #
     # It mainly setups suitable BOOTPROTO an IP related values
     def configure_as_bridge_port(device)
+      selected_interface = NetworkInterfaces.Current
       log.info("Adapt device #{device} as bridge port")
 
       # when using wicked every device which can be bridged
       # can be set to BOOTPROTO=none. No workaround with
       # BOOTPROTO=static required anymore
-      NetworkInterfaces.Edit(device)
+      if NetworkInterfaces.Edit(device)
+        NetworkInterfaces.Current["IPADDR"] = ""
+        NetworkInterfaces.Current["NETMASK"] = ""
+        NetworkInterfaces.Current["BOOTPROTO"] = "none"
+        # take out PREFIXLEN from old configuration (BNC#735109)
+        NetworkInterfaces.Current["PREFIXLEN"] = ""
 
-      NetworkInterfaces.Current["IPADDR"] = ""
-      NetworkInterfaces.Current["NETMASK"] = ""
-      NetworkInterfaces.Current["BOOTPROTO"] = "none"
-      # take out PREFIXLEN from old configuration (BNC#735109)
-      NetworkInterfaces.Current["PREFIXLEN"] = ""
+        # remove all aliases (bnc#590167)
+        aliases = NetworkInterfaces.Current["_aliases"] || {}
+        aliases.each do |alias_name, alias_ip|
+          NetworkInterfaces.DeleteAlias(device, alias_name) if alias_ip
+        end
+        NetworkInterfaces.Current["_aliases"] = {}
+
+        NetworkInterfaces.Commit
+        NetworkInterfaces.Add
 
-      # remove all aliases (bnc#590167)
-      aliases = NetworkInterfaces.Current["_aliases"] || {}
-      aliases.each do |alias_name, alias_ip|
-        NetworkInterfaces.DeleteAlias(device, alias_name) if alias_ip
```

Comment:
Sorry, I would be totally baffled as a user, and would not know what to answer. Why are we even asking?